neo4j-store/src/main/resources/AcceptanceTestSetup.kts

```diff
@@ -126,4 +129,71 @@ adminStore.atomicCreateOffer(
         )
 ).mapLeft {
     throw Exception(it.message)
-}
\ No newline at end of file
+}
+
+// For US
+
+// Plan is created with trial time, which will cause payment to happen as
+// a result of Stripe events and not due to a synchronous purchase call.
+
+job {
+    create { Region(id = "us", name = "USA") }
+}.mapLeft {
+    throw Exception(it.message)
+}
+
+adminStore.removeProductAndPricePlans("PLAN_10USD_DAY")
+        .mapLeft {
+            logger.warn(it.message)
+        }
+
+adminStore.createPlan(
+        plan = Plan(
+                id = "PLAN_10USD_DAY",
+                interval = "day",
+                trialPeriod = 4000L),      // 4 sec. trial time
+        stripeProductName = "Daily subscription plan",
+        planProduct = Product(
+                sku = "PLAN_10USD_DAY",
+                price = Price(amount = 10_00, currency = "USD"),
+                properties = mapOf(
+                        PRODUCT_CLASS.s to SUBSCRIPTION.name,
```

Comment:
Similar to another comment, product class should be MEMBERSHIP.
Payment type is SUBSCRIPTION.
If no payment type is set, then the default value is ONE TIME PAYMENT.